src/app/selectors.js

```diff
@@ -148,11 +234,31 @@ export const isLoggedIn = state =>
   state.root.controllerConnection && state.root.bakery;
 
 /**
-
+  Returns a model status for the supplied modelUUID.
+  @param {String} modelUUID The model UUID to fetch the status for
+  @returns {Function} The memoized selector to return the model status.
 */
 export const getModelStatus = modelUUID => {
   return createSelector(
     getModelStatuses,
     modelStatuses => getModelStatusByUUID(modelUUID, modelStatuses)
   );
 };
+
+/**
+  Returns the model statuses sorted by status.
+  @returns {Function} The memoized selector to return the sorted model statuses.
+*/
+export const getGroupedModelStatuses = createSelector(
```

Comment:
This is exported because it'll be used as a selector for determining the sorting order of the table in the follow-up